cookie-store/resources/cookie_helper.py

```diff
@@ -18,52 +18,56 @@
 #   Used to set cookies from an HTTP response header context.
 #
 # The response has 200 status and content-type: text/plain; charset=<charset>
-import cgi, encodings, os, re, sys, urllib
+import encodings, re
+
+from six.moves.urllib.parse import parse_qs, quote
+
+from wptserve.utils import isomorphic_decode, isomorphic_encode
 
 # NOTE: These are intentionally very lax to permit testing
-DISALLOWED_IN_COOKIE_NAME_RE = re.compile(r'[;\0-\x1f\x7f]');
-DISALLOWED_IN_HEADER_RE = re.compile(r'[\0-\x1f\x7f]');
+DISALLOWED_IN_COOKIE_NAME_RE = re.compile(r'[;\0-\x1f\x7f]')
+DISALLOWED_IN_HEADER_RE = re.compile(r'[\0-\x1f\x7f]')
```

Comment:
```suggestion
DISALLOWED_IN_COOKIE_NAME_RE = re.compile(br'[;\0-\x1f\x7f]')
DISALLOWED_IN_HEADER_RE = re.compile(br'[\0-\x1f\x7f]')
```

These look like byte ranges.